pkg/assets/handlers.go

```diff
@@ -154,6 +154,22 @@ type WebConsoleVersion struct {
 	OpenShiftVersion  string
 }
 
+var extensionPropertiesTemplate = template.Must(template.New("webConsoleExtensionProperties").Parse(`
+window.OPENSHIFT_EXTENSION_PROPERTIES = {
```

Comment:
template looks like it dumps 
` window.OPENSHIFT_EXTENSION_PROPERTIES = {"key1": "value1", "key2": "value2"}`

@spadgett I don't know the javascripty stuff, os that lhs (`window.OPENSHIFT_EXTENSION_PROPERTIES`) right?